easybuild/easyconfigs/g/GROMACS/GROMACS-5.1.2-foss-2015a-hybrid.eb

```diff
@@ -0,0 +1,42 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC, Ghent University
+# Authors::   Wiktor Jurkowski <wiktor.jurkowski@uni.lu>, Fotis Georgatos <fotis.georgatos@uni.lu>, \
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Kenneth Hoste
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-93.html
+#
+# Version 5.1.2
+# Author: Adam Huffman
+# The Francis Crick Institute
+##
+
+name = 'GROMACS'
+version = '5.1.2'
+versionsuffix = '-hybrid'
+
+homepage = 'http://www.gromacs.org'
+description = """GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = ['ftp://ftp.gromacs.org/pub/gromacs/']
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [
+    ('CMake', '3.4.1'),
+    ('libxml2', '2.9.2')
+]
+
+dependencies = [
+    ('Boost', '1.60.0'),
+    ('CUDA', '7.5.18', '', True),
```

Comment:
@valtandor should we make it clear that this GROMACS build is GPU-able by tweaking the `versionsuffix`, especially since the existing easyconfig do not include `CUDA` as a dependency? maybe add `-gpu`?